scipy-notebook/Dockerfile

```diff
@@ -50,8 +50,8 @@ RUN conda install --quiet --yes \
     # Also activate ipywidgets extension for JupyterLab
     # Check this URL for most recent compatibilities
     # https://github.com/jupyter-widgets/ipywidgets/tree/master/packages/jupyterlab-manager
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager@^1.1.0 --no-build && \
-    jupyter labextension install jupyterlab_bokeh@1.0.0 --no-build && \
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager@^2.0.0 --no-build && \
+    jupyter labextension install @bokeh/jupyter_bokeh@^2.0.0 --no-build && \
```

Comment:
If you could pin jupyter-matplotlib on the next line, we can merge this PR. I'm OK with ^ pinning instead of hard pinning for now. https://www.npmjs.com/package/jupyter-matplotlib suggests the latest is 0.7.2 which @romainx confirmed working in #1040 

```jupyter labextension install jupyter-matplotlib@^0.7.2 --no-build && \```

Reply:
Sure. I've pinned jupyter-matplotlib in the latest commit.

Reply:
Hello, to be homogeneous we should pin with `^` to be sure to install version greater or equal to `0.7.2` as suggested by @parente.
```diff
+jupyter-matplotlib@^0.7.2
-jupyter-matplotlib@0.7.2
````